templates/webpack.angular.js

```diff
@@ -60,7 +60,8 @@ module.exports = env => {
     const entryModule = `${nsWebpack.getEntryModule(appFullPath, platform)}.ts`;
     const entryPath = `.${sep}${entryModule}`;
     const entries = { bundle: entryPath };
-    if (platform === "ios") {
+    const areCoreModulesExternal = Array.isArray(env.externals) && env.externals.some(e => e.indexOf("tns-core-modules") > -1);
```

Comment:
You can extract this logic to a common function and reuse it from all `webpack.config` files 😸

Reply:
We will handle it by extracting a common/core config after the NativeScript 6.0.0 release.